### PR TITLE
HPA example. Upgrade helm-application dependency

### PR DESCRIPTION
**Pull request by jemacchi**
- HPA example. 
- Upgrade helm-application dependency.

### CODE REVIEW

**Review by danduk82 — APPROVED** [review]
:star_struck:

**Comment by groldan** [comment]
@danduk82 you beat me I was going to provide a couple improvements. Nevermind, will create a seprate pr

**Review by groldan — APPROVED** [review]
* Made some improvements to the HPA example's README. View it rendered here: https://github.com/camptocamp/helm-geoserver-cloud/blob/hpa/examples/pgconfig-wms-hpa/README.md
* fixed a couple mispelled commands
* changed the stress script to use Apache Benchmark

File: examples/pgconfig-wms-hpa/README.md [hunk 1]
```diff
@@ -49,40 +50,153 @@ gs-cloud-pgconfig-wms-hpa-gsc-wms-758dfd8765-qs946       1/1     Running   0
...
+awk '{printf("%s\t%s\n", $4, $3)}' | sudo tee -a /etc/hosts
+```
+
+
+### 4. Initialize the catalog:
+
+Run the following command to create a test layer:
+
+```shell
+/examples/pgconfig-wms-hpa/init-catalog.sh
```
Comment (danduk82) [inline examples/pgconfig-wms-hpa/README.md:73]: the path here should be relative, it's missing a `.` character, so it should be 
```suggestion
./examples/pgconfig-wms-hpa/init-catalog.sh
```

**Review by danduk82 — CHANGES_REQUESTED** [review]
I found just one little typo in the docs

File: examples/pgconfig-wms-hpa/README.md (hunk 1 above)
Comment (danduk82) [reply examples/pgconfig-wms-hpa/README.md:73]: ping @groldan

**Review by jemacchi — COMMENTED** [review]
LGTM

**Review by groldan — APPROVED** [review]

**Review by danduk82 — APPROVED** [review]

**Pull request merged**